manifests/a/Acronis/CyberProtectHomeOffice/26.1.39703/Acronis.CyberProtectHomeOffice.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-2
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Acronis.CyberProtectHomeOffice
+PackageVersion: 26.1.39703
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallerSwitches:
+  Silent: -silent_installation
+  SilentWithProgress: -silent_installation #-start_installation
+  Log: -log:"<LOGPATH>"
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://dl.acronis.com/u/AcronisCyberProtectHomeOffice.exe
+  InstallerSha256: 136F1D43F363097B8F85FC06B56EA045D57610AAD43268F129D388C551D26514
```

This package may require AppsAndFeaturesEntries to be able to upgrade -
```suggestion
  InstallerSha256: 136F1D43F363097B8F85FC06B56EA045D57610AAD43268F129D388C551D26514
  Scope: machine
  AppsAndFeaturesEntries: 
  - DisplayName: Acronis Cyber Protect Home Office
    DisplayVersion: 26.1.39703
    InstallerType: msi
    ProductCode: '{E1FEA3C4-5790-465E-ADDD-6FF1E9FDCAB4}'
  - DisplayName: Acronis Cyber Protect Home Office
    DisplayVersion: 26.1.39703
    InstallerType: exe
    ProductCode: '{E1FEA3C4-5790-465E-ADDD-6FF1E9FDCAB4}Visible'
  - DisplayName: Acronis Drivers
    DisplayVersion: 26.1.39703
    InstallerType: msi
    ProductCode: '{C5293FA2-D7C1-4445-9DC9-3E34349D59AF}'
```